app/src/pages/tracing/TracesTable.tsx

```diff
@@ -144,7 +144,7 @@ export function TracesTable(props: TracesTableProps) {
                 descendants {
                   spanKind
                   name
-                  statusCode
+                  propagatedStatusCode
```

Comment:
use alias here too

Reply:
addressed in [c4f88f4](https://github.com/Arize-ai/phoenix/pull/1824/commits/c4f88f45f6b6b15167a7284e8484e93340e4b1ac)